libraries/AP_GPS/AP_GPS_SBP.cpp

```diff
@@ -289,6 +289,7 @@ AP_GPS_SBP::_attempt_state_update()
 
         last_full_update_tow = last_vel_ned.tow;
         last_full_update_cpu_ms = now;
+        state.rtk_age_ms        = last_full_update_cpu_ms - last_injected_data_ms;
```

Comment:
@WickedShell I did not get what you meant with https://github.com/amilcarlucas/ardupilot/blob/7af8de19a0d93030e3926695346f85d40bf275e6/libraries/AP_GPS/AP_GPS_SBP.cpp#L405

I'm using 'last_injected_data_ms' here already